R/landing_popup_module.R

```diff
@@ -0,0 +1,118 @@
+#' Landing Popup Module
+#'
+#' @description This module creates a landing welcome popup for `teal` applications.
+#'
+#' @details If you use this module in `teal::init(modules = )`, it will not be wrapped in a tab in `teal` application.
+#'
+#' @param label `character(1)` the label of the module.
+#' @param title `character(1)` the text to be displayed as a title of the popup.
+#' @param content The content of the popup. Passed to `...` of `shiny::modalDialog`. Can be a `character`
+#' or a text input control (like `textInput`) or a list of `shiny` tags. See examples.
```

Comment:
Only text input control? In principle one could put anything in.

Reply:
I think I will remove this `text input control` and will just leave `list of shiny tag` : )

Reply:
0179152845904a5208c4c1306da62eee4481966e